src/withdraws/Withdrawer.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {BaseVault} from "src/BaseVault.sol";
+import {ISlisBnbStakeManager} from "src/interface/external/lista/ISlisBnbStakeManager.sol";
+import {MainnetContracts} from "script/Contracts.sol";
```

Comment:
i have a concern about this MainnetContracts

when you pull it into a dependency it reads all the variables in there and also increases contract size, even though you use 1,2 of them

maybe that is mitigated automatically if we have the logic of `_asyncWithdrawBalance` in a separate external library?

Reply:
Yeah we could def move this logic to a separate library or contract